manifests/controller-manager/cloud-controller-manager-role-bindings.yaml

```diff
@@ -1,5 +1,22 @@
 apiVersion: v1
 items:
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: servicecatalog.k8s.io:apiserver-authentication-reader
+    namespace: kube-system
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: extension-apiserver-authentication-reader
+  subjects:
+  - apiGroup: ""
+    kind: ServiceAccount
+    name: cloud-controller-manager
+    namespace: kube-system
+  - apiGroup: ""
+    kind: User
+    name: cloud-controller-manager
```

Comment:
This is as a result of the breaking change introduced with #228 right?

Reply:
I believe it is... but I didn't 100% verify that.